common/sgx/revocation.c

```diff
@@ -240,13 +238,7 @@ oe_result_t oe_enforce_revocation(
     // constraint. If the crl_issuer_chain was different from the certificate
     // chain, then verification would fail because the CRLs will not be found
     // for certificates in the chain.
-    r = oe_cert_verify(
-        leaf_cert, &crl_issuer_chain[0], crl_ptrs, 2, &cert_verify_error);
-    if (r != OE_OK)
-    {
-        OE_RAISE_MSG(
-            r, "oe_cer_verify failed with error = %s\n", cert_verify_error.buf);
-    }
+    OE_CHECK(oe_cert_verify(leaf_cert, &crl_issuer_chain[0], crl_ptrs, 2));
```

Comment:
I know you didn't introduce it, but could you use **OE_COUNTOF(crl_pts)** instead the hard-coded **2**?

Also, I think **crl_issuer_chain** is clearer than **&crl_issuer_chain[0]**.